docs/getting_started.md

```diff
@@ -203,6 +203,136 @@ See these resources for more information on clustering:
  - [Quantifying clustering behavior in Orchestrating Single Cell Analysis](https://bioconductor.org/books/release/OSCA.advanced/clustering-redux.html#quantifying-clustering-behavior)
 
 
+### Cell type annotation
+
+Processed objects may contain cell type annotations and associated metadata from one or more of the following sources.
+
+- Submitter-provided annotations
+  - Note that these are only present for a subset of libraries.
+- Annotations determined by [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html), an automated reference-based method ([Looney _et al._ 2019](https://doi.org/10.1038/s41590-018-0276-y)).
+- Annotations determined by [`CellAssign`](https://github.com/Irrationone/cellassign), an automated marker-gene based method ([Zhang _et al._ 2019](https://doi.org/10.1038/s41592-019-0529-1)).
+
+If at least one type of cell type annotation is present, a supplemental cell type report will be provided with the download.
```

Comment:
```suggestion
If at least one method was used for cell type annotation, a supplemental cell type report will be provided with the download.
```
I feel like we want to use the word "method" rather than "type" when referring to what kind of cell type annotations are present. I know submitter isn't really a method, but the other two are... So if you have a better word I'm open to other options!

Reply:
honestly i think we are doomed with this wording no matter what, so i'm ok with it!